packages/css/src/field.css

```
@@ -14,7 +14,7 @@
   /**
    * States
    */
-  &:has([aria-disabled='true'], :disabled) > * {
+  &:has([aria-disabled='true'], :disabled:not(select option)) > * {
```

Do we need to specify `select option` - it can be just `option` as a `:disabled` option in `datalist` should not disable as well? ☺️ 
```suggestion
  &:has([aria-disabled='true']:not(u-option), :disabled:not(option)) > * {
```

---

I originally did that, but decided to be more specific, since we have changed some CSS to be more specific other places. This makes in consistent with that.

---

Mm, understand that, but I think this is not a case "to be consistent". Where we have done specific selectors is because we want to enable the user to nest tags without getting accidental styling. Here on the other hand, we want to enable infinite nesting, but never mistakenly disable a field ☺️  I think we should add:
```
 &:has([aria-disabled='true']:not(u-option), :disabled:not(option)) > * {
```

---

I updated the PR, since `option` in a group would break this. Your suggestion has less edgecases that I can think of

---

A point is also that `option` is not the "true" element used in field - so it should never have anything to do with `disabled`